modules/test/playwright/tests/export-import-web/main/stagingUseCase.spec.ts

```diff
@@ -255,7 +256,7 @@ test(
 );
 
 test(
-	'Cannot publish if linked file does not exist',
+	'cannot publish if linked file does not exist',
```

Comment:
I though the unification was to do all starting uppercase 🤔 At least it's how it was in Sandro's example [here](https://github.com/liferay-page-management/liferay-portal/pull/15970/files#diff-f65b3fb1f8384b0799736f4508879e7db12a6649d3cf54bcee26741f0ac46e77)